scripts/report_pytest_markers.py

```
@@ -0,0 +1,436 @@
+# SPDX-FileCopyrightText: Copyright (c) 2024-2025 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
```

Ruff (0.14.8)

The file is executable but no shebang is present

(EXE002)